apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -102,23 +102,18 @@ private static Patient transform(Beneficiary beneficiary, RequestHeaders request
 
     // Required values not directly mapped
     patient.getMeta().addProfile(ProfileConstants.C4BB_PATIENT_URL);
-
     patient.setId(beneficiary.getBeneficiaryId());
-    Optional<String> mbiUnhashedCurrent = beneficiary.getMedicareBeneficiaryId();
 
+    // BENE_ID => patient.identifier
     TransformerUtilsV2.addIdentifierSlice(
         patient,
         TransformerUtilsV2.createCodeableConcept(
```

Comment:
For consistency, I'd set the display on this type coding for `MB`. It currently isn't set here but it is for the `MC` identifiers:

![image](https://user-images.githubusercontent.com/29063436/115896165-beaf4b80-a420-11eb-8d04-be17c0d55384.png)

vs

![image](https://user-images.githubusercontent.com/29063436/115896196-c5d65980-a420-11eb-9f65-c3a2755a31a5.png)

Reply:
Seeing the display in there now after the most recent changes 👍